Modules/sre_lib.h

```diff
@@ -1200,10 +1200,10 @@ SRE(match)(SRE_STATE* state, const SRE_CODE* pattern, int toplevel)
 
         case SRE_OP_POSSESSIVE_REPEAT:
             /* create possessive repeat contexts. */
-            /* <POSSESSIVE_REPEAT> <skip> <1=min> <2=max> pattern
```

Comment:
What about POSSESSIVE_REPEAT? `repeat_index` is not used here. MAX_UNTIL and MIN_UNTIL are not used with POSSESSIVE_REPEAT, so there is no problem with keeping POSSESSIVE_REPEAT intact.

Reply:
If I add more conditional checks, I'm afraid to slow down the pattern compilation.

Reply:
Not much. Checking `if op is POSSESSIVE_REPEAT` is cheap in comparison with parsing or optimization.